scripts/exregional_make_grid.sh

```diff
@@ -267,33 +267,28 @@ generation executable (exec_fp):
 #
   settings="
 'regional_grid_nml':
-    'plon': ${LON_CTR}
-    'plat': ${LAT_CTR}
-    'delx': ${DEL_ANGLE_X_SG}
-    'dely': ${DEL_ANGLE_Y_SG}
-    'lx': ${NEG_NX_OF_DOM_WITH_WIDE_HALO}
-    'ly': ${NEG_NY_OF_DOM_WITH_WIDE_HALO}
-    'pazi': ${PAZI}
+  'plon': ${LON_CTR}
+  'plat': ${LAT_CTR}
+  'delx': ${DEL_ANGLE_X_SG}
+  'dely': ${DEL_ANGLE_Y_SG}
+  'lx': ${NEG_NX_OF_DOM_WITH_WIDE_HALO}
+  'ly': ${NEG_NY_OF_DOM_WITH_WIDE_HALO}
+  'pazi': ${PAZI}
 "
 
-  # Store the settings in a temporary file
-  tmpfile=$( $READLINK -f "$(mktemp ./namelist_settings.XXXXXX.yaml)")
-  cat > $tmpfile << EOF
+  (cat << EOF
 $settings
 EOF
-
-  uw config realize \
-    -i ${tmpfile} \
+) | uw config realize \
```

Comment:
Just FYI, since a `settings` variable is defined (unlike in my previous example, which had no intermediate variable), I believe you can also just:
```
echo "$settings" | uw config realize ...
```
without the `( cat ... )` subshell command, e.g.
```
$ export VAR1=foo
$ export VAR2=bar
$ settings="
config:
  foo: $VAR1
  bar: $VAR2
"
$ echo "$settings" | uw config realize --input-format yaml --output-format nml
&config
    foo = 'foo'
    bar = 'bar'
/
```